BroadcomCloudSwg/connectors/broadcom_cloud_swg_connector.py

```diff
@@ -231,11 +241,13 @@ async def get_events(self) -> tuple[int, datetime]:
                                 line_as_dict = BroadcomCloudSwgClient.parse_input_string(line, headers)
 
                                 line_date_time = BroadcomCloudSwgClient.get_date_time_from_data(line_as_dict)
-                                if latest_date is None:  # pragma: no cover
-                                    latest_date = line_date_time
 
-                                if line_date_time and latest_date and latest_date < line_date_time:
-                                    latest_date = line_date_time
+                                if line_date_time:  # pragma: no cover
+                                    if line_date_time < previous_latest_date:
+                                        continue
+
+                                    if latest_date < line_date_time:
+                                        latest_date = line_date_time
```

Comment:
I believe it's normal having some events running a little late in the downloaded archive, because the events took time to be collected by broadcom.

Reply:
Make sense. Then I'll remove the statement about continuation of process